rustls/src/conn.rs

```diff
@@ -478,11 +478,27 @@ impl<Data> ConnectionCommon<Data> {
                 return Ok((rdlen, wrlen));
             }
 
-            if !eof && self.wants_read() {
-                match self.read_tls(io)? {
-                    0 => eof = true,
-                    n => rdlen += n,
-                }
+             while !eof && self.wants_read() {
+                debug!("read_tls");
+                match self.read_tls(io) {
+                    Ok(0) => {
+                        eof = true;
+                        debug!("eof");
+                        break;
+                    }
+                    Ok(n) => {
+                        rdlen += n;
+                        debug!("read {} bytes", n);
```

Comment:
I think we should remove the `debug!()` call here, as that seems pretty noisy and not that useful?